src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.ts

```diff
@@ -154,4 +154,10 @@ export class BootstrapCreateModalComponent implements OnDestroy, OnInit {
     });
     taskObs.subscribe(undefined, finishHandler, finishHandler);
   }
+
+  copyToClipBoard(token: any) {
+    token.select();
+    document.execCommand('copy');
+    token.setSelectionRange(0, 0);
+  }
```

Comment:
I would rather use the [Web Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API): `Document.execCommand` is deprecated and could be removed anytime soon (some [sample code](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard)).

I was going to recommend you using a ready-to-use 3rd-party package (like [this](https://www.npmjs.com/package/ngx-clipboard)), but then checked the code and they're using 'execCommand' too :laughing: Given the size of this snippet, I think it doesn't mind if we don't use an external package.

Reply:
I updated the previous clipboard button and used it.